denys/Sunshine/S01.01-Exercise-CreateLayout/app/src/main/res/layout/activity_forecast.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+     Copyright (C) 2016 The Android Open Source Project
+
+     Licensed under the Apache License, Version 2.0 (the "License");
+     you may not use this file except in compliance with the License.
+     You may obtain a copy of the License at
+
+          http://www.apache.org/licenses/LICENSE-2.0
+
+     Unless required by applicable law or agreed to in writing, software
+     distributed under the License is distributed on an "AS IS" BASIS,
+     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+     See the License for the specific language governing permissions and
+     limitations under the License.
+-->
+<!--TODO (1) Change the ConstraintLayout to a FrameLayout-->
+<!--TODO (7) Remove the line that declares the id, we don't need it-->
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <!--TODO (8) Wrap the TextView in a ScrollView-->
+    <!--TODO (9) Set the ScrollView width to match_parent and the height to wrap_content-->
+
+    <!--TODO (2) Change the TextView's android:id attribute to tv_weather_data-->
+    <!--TODO (3) Give the TextView 16dp of padding-->
+    <!--TODO (4) Set the text size to 20sp-->
+    <!--TODO (5) Remove all attributes with the word constraint in them-->
+    <!--TODO (6) Remove the default text from the TextView-->
+    <ScrollView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <TextView
+            android:id="@+id/tv_weather_data"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:padding="16dp"
+            android:textSize="20sp" />
```

Comment:
Just in case you forgot: consider moving these values to the app resources (xml files in the `res` directory) :)